### PR TITLE
chore(avatars): Change avatar initials to use approved colors

### PR DESCRIPTION
**Pull request by patrickpaul**
Per Box Design, the following changes were requested to `<AvatarInitials>`:

Force initials to be uppercase
Change font-weight from default (400) to `bold`
Change colors to use the approved list:
Avatar Replacement Colors

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @patrickpaul has signed the CLA. Thanks for the pull request!

**Comment by booc0mtaco** [comment]
Can we reference the generated variables at https://github.com/box/box-ui-elements/blob/master/src/styles/variables.js (and add the missing ones to the SCSS file)

File: src/components/avatar/AvatarInitials.js [hunk 1]
```diff
@@ -2,15 +2,15 @@
...
-    '#18BBF7',
-    '#0D67C7',
-    '#052E5C',
-    '#747679',
-    '#FDA308',
-    '#98C332',
-    '#159F45',
-    '#B800B2',
-    '#F22C44',
+    '#2486FC',
```
Comment (priyajeet) [inline src/components/avatar/AvatarInitials.js]: can these come from styles/variables?

**Comment by priyajeet** [comment]
Light blue and box blue look kinda similar

**Comment by alexkrolick** [comment]
Might have some small conflicts withhttps://github.com/box/box-ui-elements/pull/1059 but should be ok

**Comment by alexkrolick** [comment]
#1076 incorporated into this PR that also refactors the avatars

**Pull request closed without merging**